vip-feed-cache.php

```diff
@@ -0,0 +1,99 @@
+<?php
+// Require core feed class.
+require_once( ABSPATH . WPINC . '/class-feed.php' );
```

Comment:
Rather than loading `class-feed.php` (and as a result, SimplePie) on every single request, we should move this require + our custom classes to a separate file, which is then `require`-ed inside `vipgo_feed_options`